modules/test/playwright/tests/headless-builder-web/schemas.spec.ts

```diff
@@ -612,21 +608,6 @@ testFeatureFlagsEnabled(
 				.getByText('Author')
 		).not.toHaveClass(/disabled/);
 
-		// Assert that modifiable system object properties are enabled
```

Comment:
Is the API Application not going to be in the list ever? I don't fully understand why :thinking:

Reply:
@ccorreagg this is a question for product :D A thread here https://liferay.slack.com/archives/C03U66A6NNN/p1729156298128819

Reply:
the // Assert that modifiable system object properties are enabled keeps passing in CI, but locally, if I don't have the FF enabled in properties, it fails